scripts/exrrfs_jedivar.sh

```diff
@@ -54,7 +57,32 @@ cp "${UMBRELLA_PREP_IC_DATA}"/*satbias* satbias_in/.
 #
 # copy observations files
 #
-cp "${COMOUT}"/ioda_bufr/det/* obs/.
+obspath_bufr="${COMOUT}/ioda_bufr/${WGF}"
+obs_files_source[0]="${obspath_bufr}"/ioda_adpupa.nc
+obs_files_target[0]=ioda_adpupa.nc
+
+obs_number=${#obs_files_source[@]}
```

Comment:
Suggest adding a comment after line 64: 
`# this increment obs_number by 1`

Reply:
Done